lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,11 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.next = (dow) => {
+  const dows = ['Sunday','Monday','Tuesday',  'Wednesday','Thursday', "Friday",'Saturday'];
+  let days = (dows.indexOf(dow)-(new Date()).getDay());
+  if (days < 0) days+=7;
+  if (dows.includes(dow)) return new Date((new Date()).getFullYear(), (new Date()).getMonth(), (new Date()).getDate() + days);
```

Comment:
This is a really complex single line.  It would be better if it conformed to the style guide, but it would also be helpful to just say `new Date()` once and then reuse that.  For example, something like this:

```js
if (dows.includes(dow)) {
  const today = new Date();
  return new Date(today.getFullYear(), today.getMonth(), today.getDate() + days);
}
```